### DISCUSSION TITLE
Make Docker container work the same as npm how-to

### DISCUSSION THREAD

**Discussion by officerNordberg** [CycloneDX/cyclonedx-node-module#315] [Ideas]
I tried to use  cyclonedx/cyclonedx-node in a pipeline and to my surprise 

`/bin/sh: eval: line 44: cyclonedx-node: not found`

I had to use  /usr/src/cyclonedx-bom/bin/make-bom.js instead. Feels like the container name should match the command and global install

**Comment by jkowalleck** [comment]
There is no need to execute a command inside the container. The docker entry point is already the command you are looking for.
please follow the instructions in the readme: 

> Run `cyclonedx/cyclonedx-node` docker image inside your project folder using:
> 
> ```shell
> docker run --rm \
>   -v "$PWD":/src \
>   -w /src \
>   cyclonedx/cyclonedx-node -o /src/bom.xml
> ```

**Comment by jkowalleck** [comment]
this conversation got closed due to #321
